msg/ja_JP.utf8/cubrid.msg

```diff
@@ -1352,8 +1352,10 @@ $ LOADDB
 1281 Error reserved for scalability development.
 1282 Error reserved for scalability development.
 1283 Error reserved for scalability development.
+1284 %1$s sで参照されている主キーが原因で、テーブルを切り捨てることができません。 CASCADEオプションを使用して再試行してください。
```

Comment:
> When I reviewed in the 'Files changed' tab, some characters look like broken characters. But, the followings show well. I don't know why.

I don't know why the "Files Changed" show in a wrong way, but I think files are correct because we can see the not-broken file in a vi, bash, and even here. It may be due to the encoding of github. What do you think?